grafeas-client/.repo-metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "grafeas-client",
```

Comment:
The `name` in other multi-word gems is usually smashed: `recaptchaenterprise`.

Reply:
Yeah good catch. This is probably supposed to be `"grafeas"`, the API name.